tools/src/dbgen/Lib.hs

```diff
@@ -30,38 +35,55 @@ import           Pos.Wallet.Web.ClientTypes
 import           Pos.Wallet.Web.ClientTypes.Instances ()
 import           Pos.Wallet.Web.Methods.Logic
 import           Pos.Wallet.Web.Methods.Restore
+import           Pos.Wallet.Web.State.State           (getWalletUtxo, setWalletUtxo, updateWalletBalancesAndUtxo)
+import           Test.QuickCheck                      (arbitrary, generate)
 import           Rendering
 import           Text.Printf
 
 --
 -- Types
 --
 
-data GenSpec = GenSpec {
-    wallets       :: !Integer
+data GenSpec = GenSpec
+    { wallets     :: !Integer
     -- ^ How many wallets to create
     , wallet_spec :: WalletSpec
     -- ^ The specification for each wallet.
+    , fakeUtxo    :: FakeUtxoSpec
```

Comment:
It probably makes more sense to make the FakeUtxoSpec nested into a WalletSpec. This will match my suggestion above (of the Dhall config changes).